strapi/src/admin/app.example.tsx

```diff
@@ -0,0 +1,35 @@
+export default {
+  config: {
+    locales: [
+      // 'ar',
+      // 'fr',
```

Comment:
```suggestion
      'fr',
```

Reply:
Good point! Not sure yet if we'll enable the locales at runtime or have them statically set. Let's keep this in mind for when we implement locales in front-ends 👍